src/web/client/test/integration/urlBuilderUtil.test.ts

```diff
@@ -23,9 +23,10 @@ import {
 import { httpMethod } from "../../common/constants";
 import * as schemaHelper from "../../utilities/schemaHelperUtil";
 
-import PowerPlatformExtensionContextManager, {
+import PowerPlatformExtensionContext, {
```

Comment:
Should we rename this to "PowerPlatformWebExtensionContext"?

Reply:
Naming is a tough business - existing one was too long - any shorter suggestions?